Build failure

There is error during build of the latest 1.8 version when package downloaded from PyPI:
```
      lib/cyac/xstring.cpp:1113:10: fatal error: 'unicode_portability.c' file not found
      #include "unicode_portability.c"
               ^~~~~~~~~~~~~~~~~~~~~~~
      1 error generated.
      error: command '/usr/bin/clang' failed with exit code 1
```

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -5,4 +5,4 @@ global-exclude trie.cpp
 global-exclude utf8.cpp
 global-exclude xstring.cpp
 global-exclude __pycache__
-global-include unicode_portability.cpp
+global-include unicode_portability.c

